corehq/apps/hqwebapp/static/hqwebapp/js/ui_elements/ui-element-key-val-list.js

```diff
@@ -34,12 +35,12 @@ hqDefine('hqwebapp/js/ui_elements/ui-element-key-val-list', [
         var $modalContent = $('<div class="modal-content" />');
 
         $modalContent.prepend('<div class="modal-header"><a class="close" data-dismiss="modal">×</a><h4 class="modal-title">'
-            + django.gettext('Edit Mapping for ') + this.modal_title + '</h4></div>');
+            + django.gettext('Edit ') + this.modal_title + '</h4></div>');
```

Comment:
Could you make "Edit" part of the modal title? Combining translated strings doesn't allow the translator to re-order the words if needed.